src/selectors/ProfilePageSelector.js

```diff
@@ -164,12 +165,21 @@ export const handleSaveProfileSelector = createSelector(
   }),
 );
 
+export const handleFetchProfileSelector = createSelector(
+  isCurrentUserProfileSelector,
+  userAccountSelector,
+  (isCurrentUserProfile, userAccount) => ({
+    isCurrentUserProfile,
+    userAccount,
+  }),
+);
+
 // Reformats the social links in a platform-keyed hash.
 const socialLinksByPlatformSelector = createSelector(
   profileAccountSelector,
   (account) => {
     const linksByPlatform = {};
-    if (account.socialLinks !== undefined) {
+    if (account.socialLinks != null) {
```

Comment:
The `userAccount` reducer has different defaults for socialLinks, so this needed to be made a bit less stringent, along with the one below.  In our profile reducer, we default it to an empty array, whereas in userAccount it's just null.

Reply:
Should it become a function?  `hasSocialLinks` or similar?

Reply:
Eh... could be, but I don't think it'll see much reuse or anything.

Reply:
Actually, I'm going to change it to use `Array.isArray()` - that'll be more specific to what this is trying to check for.

Reply:
I thought you had to change it in two places in this PR-- or is line 195 a different object?